stdlib/builtin.rbi

```diff
@@ -273,6 +278,7 @@ class Numeric
 end
 
 class Integer < Numeric
+  def to_i: -> Integer
```

Comment:
Removing `Object#to_i` causes the following error:

```
Unexpected error found: smoke/map/a.rb:5:15: NoMethodError: type=::Integer, method=to_i (x.to_i)
```

So, `Integer#to_i` must be added. `to_i` is an alias for `to_int`.
See <http://ruby-doc.org/core-2.5.3/Integer.html#method-i-to_i>.